From: Modestas Vainius <modax@debian.org>
Subject: Fix FTBFS with gcc 4.7
Origin: vendor

Fixes the following FTBFS with gcc 4.7:

../../generator/reporthandler.cpp: In static member function 'static void ReportHandler::warning(const QString&)':
../../generator/reporthandler.cpp:60:62: error: format not a string literal and no format arguments [-Werror=format-security]
cc1plus: some warnings being treated as errors
make[4]: *** [release/reporthandler.o] Error 1


--- generator/reporthandler.cpp
+++ generator/reporthandler.cpp
@@ -57,7 +57,7 @@ void ReportHandler::warning(const QStrin
     if (db && db->isSuppressedWarning(warningText)) {
         ++m_suppressed_count;
     } else if (!m_reported_warnings.contains(warningText)) {
-        qDebug(qPrintable(warningText));
+        qDebug() << warningText;
         ++m_warning_count;
 
         m_reported_warnings.insert(warningText);
